src/main/java/com/google/cloud/tools/appengine/api/debug/GenRepoInfoFile.java

```diff
@@ -17,9 +17,49 @@
 package com.google.cloud.tools.appengine.api.debug;
 
 import com.google.cloud.tools.appengine.api.AppEngineException;
+import com.google.cloud.tools.appengine.cloudsdk.CloudSdk;
+import com.google.cloud.tools.appengine.cloudsdk.GcloudRunner;
+import com.google.cloud.tools.appengine.cloudsdk.internal.args.GcloudArgs;
+import com.google.cloud.tools.appengine.cloudsdk.process.ProcessHandlerException;
+import com.google.common.base.Preconditions;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
-/** Main interface for generating source context files, needed by the Cloud Debugger. */
-public interface GenRepoInfoFile {
+/** Interface for generating source context files using gcloud, needed by the Cloud Debugger. */
```

Comment:
Do you want to use the term "Interface"?

Reply:
I did, but I don't think my choice of words was great, will change it.